### PR TITLE
[INLONG-2050] Audit Store for Elasticsearch

### LINKED ISSUE #2050: [Feature] [InLong audit] Audit Strore for Elasticsearch
### Description

This is a sub issue for releasing InLong AuditDds of Elasticsearch.
image

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pocozh**
### Title Name: [INLONG-2050][InLong audit] Audit Strore for Elasticsearch

fix #2050

### CODE REVIEW

File: inlong-audit/audit-store/conf/application.properties [hunk 1]
```diff
@@ -48,7 +48,27 @@ mybatis.type-aliases-package=org.apache.inlong.store.entities
...
 audit.config.manager.server.url=http://127.0.0.1:8000
+# store.server: elasticsearch / mysql
+audit.config.store.server=elasticsearch
 
 # pulsar config
 audit.pulsar.server.url=pulsar://127.0.0.1:6650
-audit.pulsar.topic=persistent://public/default/audit
\ No newline at end of file
+audit.pulsar.topic=persistent://public/default/audit
+audit.pulsar.consumer.sub.name=test
```
Comment (dockerzhang) [inline inlong-audit/audit-store/conf/application.properties:57]: it's better to use the name with the 'audit' keyword.

**Review by dockerzhang — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]
+1

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/entities/ESDataPo.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+package org.apache.inlong.store.db.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class ESDataPo {
```
Comment (doleyzi) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/entities/ESDataPo.java:27]: ESDataPo miss packetId information,it need add packetId to deduplicate

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/entities/ESDataPo.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    private Date logTs;
+    private String inlongGroupId;
+    private String inlongStreamId;
+    private String auditId;
+    private long count;
+    private long size;
+    private long delay;
+
+    public String getDocId() {
+        String docId = ip + dockerId + threadId + logTs + inlongGroupId + inlongStreamId + auditId + size;
```
Comment (doleyzi) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/entities/ESDataPo.java:41]: String docId = ip + dockerId + threadId +sdkTs+ packetId + logTs + inlongGroupId + inlongStreamId + auditId + count;

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**